CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
--
+- **cf-typography:** [PATCH] Add em dash to pull quote citation.
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/517/commits/488321c27b27ce068e092201c3f278a8ef15aa86